resources/serverless/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       {{- end }}
       {{- end }}
     spec:
-      serviceAccountName: {{ template "fullname" . }}
+      serviceAccountName: {{ template "fullname" . }}-controller-manager
```

Comment:
If you changed serviceAccount name, please also change in `values.yaml` - this name is also used by Job for building Function image.